ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json

```diff
@@ -2287,5 +2287,12 @@
     "CaseFieldID": "draftOrder10Document",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "additionalApplicationsBundle",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "RU"
```

Comment:
is it not only R?

Reply:
same as above. supportingDocumentsBundle field has `U` permissions so, the parent element should have `U` as well?